adapter/internal/adapter/adapter.go

```diff
@@ -254,6 +254,12 @@ func Run(conf *config.Config) {
 		go restserver.StartRestServer(conf)
 	}
 
+	// Proxy server for the LA
+	if true {
```

Comment:
This if condition has no meaning right?  Apart from that this is just a log only. I guess we need to start the server explicitly.

Reply:
Yes Viraj..
This is a mistake.